Makefile

```diff
@@ -7,3 +10,11 @@ protobuf:
 	cd Tests/Protobuf; \
 	protoc test_scenarios.proto --swift_out=../CombineGRPCTests/Generated/; \
 	protoc test_scenarios.proto --grpc-swift_out=../CombineGRPCTests/Generated/
+
+protobuf_docker:
+	mkdir -p Tests/CombineGRPCTests/Generated
+
+	docker run --rm -v "${PWD}":/protobuf -w="/protobuf" cyborgthefirst/grpc-swift:1.0.0-alpha.9 \
```

Comment:
Will you be publishing updated images as the gRPC Swift project publish new releases?

Reply:
Yes, I use gRPC-swift in another project at work, where I will need to update these images anyway.